gapic/src/main/com/google/gapid/perfetto/models/CounterInfo.java

```diff
@@ -188,4 +190,16 @@ public static Type of(String string) {
       }
     }
   }
+
+  public static enum Interpolation {
+    Delta, // the value represents the counter "amount" since the last sample.
+    Event; // the value represents the current value until the next sample.
+
+    public static Interpolation of(QueryEngine.Row row) {
+      // Only GPU counters, and not the gpufreq counter, are Delta counters.
+      // TODO: this should be part of the counter definition in the backend.
```

Comment:
Could you please file a bug and add the id here?

Reply:
Oops, missed this before submitting. Filed bug: http://b/148458673